lib/src/test/java/com/thenewboston/utils/Mocks.kt

```diff
@@ -109,5 +111,22 @@ object Mocks {
         trust = 100.0
     )
 
+    fun invalidBlock() = InvalidBlockDTO(
```

Comment:
Can we give these "String" values meaningful values? I know it's a mock, but it's good practice to match the real response.

Reply:
no worries